services/CurlingEventService.js

```diff
@@ -368,6 +368,100 @@ class CurlingEventService {
       throw error;
     }
   }
+
+  async createTeam(name, affiliation, note) {
+    const values = [name, affiliation, note];
+    try {
+      const data = await this.#pool
+        .query(Queries.CREATE_TEAM, values);
+      return data.rows;
+    }
+    catch (err) {
+      if (err.message.includes('affiliation')) {
+        throw Exceptions.insertionException('Invalid affiliation/organization id.');
+      }
+    }
+  }
+
+  async updateTeam(id, name, affiliation, note) {
+    if (affiliation === null || affiliation === undefined) {
+      const events = await this.#pool.query(Queries.GET_FRIENDLY_EVENTS_BY_TEAM, [id]);
+      if (events.rowCount > 0) {
```

Comment:
events.RowCount is for the retrieved rows when processing the query and not the number of rows in the response. Use events.rows.length to check how many rows were output